src/db.rs

```diff
@@ -95,11 +97,14 @@ pub async fn list_all_hosts(db: &Client) -> Result<Vec<HostSummary>> {
         .database("host_statistics")
         .collection("holoports_status");
 
+    let cutoff_ms = get_cutoff_timestamp(cutoff);
+    println!("{}", cutoff_ms.to_string());
     let pipeline = vec![
         doc! {
-            // only successful ssh results
+            // only successful ssh results in last 7 days
```

Comment:
again, 7 days is an arbitrary number, while there is a number of days that you are passing to API via query parameter